cmake/order_paths.py

```diff
@@ -18,6 +18,12 @@ def main():
     parser.add_argument('--prefixes', nargs='*', help='The semicolon-separated prefixes defining the order')
     args = parser.parse_args()
 
+    # resolve the source space if any
+    spaces = []
+    for prefix in args.prefixes:
+        spaces += [prefix]
+        spaces += get_spaces([prefix])
```

Comment:
This nee variable isn't being used anywhere?

This also makes me believe that the patch wasn't actually tested.

Reply:
Sorry I wasn't retesting it again on the pull request branch. I will push a new commit to address it soon.